cpu/sam0_common/periph/rtc_rtt.c

```diff
@@ -135,13 +135,25 @@ static inline void _rtc_set_enabled(bool on)
 
 static inline void _rtt_reset(void)
 {
+#if CONFIG_SAM0_PERIPH_RTT_RTC_PERSIST_GPR
+#ifdef RTC_GPR_NUM
+    uint32_t persist[RTC_GPR_NUM];
+    int i;
+    for (i = 0; i < RTC_GPR_NUM; i++) persist[i] = RTC->MODE0.GP[i].reg;
```

Comment:
```suggestion
    for (unsigned i = 0; i < RTC_GPR_NUM; i++) {
        persist[i] = RTC->MODE0.GP[i].reg;
    }
```

No need to squish that all on one line :wink: